app/models/user.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+class User < ApplicationRecord
+  ADMIN = 'admin'
+  CUSTOMER = 'customer'
+
+  ROLES = [ADMIN, CUSTOMER].freeze
```

Comment:
constants :+1: 

would recommend using a hash instead of an array for enum: that will make it easier to manage roles in the future. imagine, you need to split the role `customer` to `reviewer`, `salesman` and `bookworm`. the change `ROLES = [ADMIN, REVIEWER, SALESMAN, BOOKWORM]` would mean that each customer receives the access lvl of the reviewer until a direct data manipulation

```
enum role: {
  admin: ADMIN = 'admin',
  customer: CUSTOMER = 'customer'
}
```
can prevent such kind of data inconsistency